pkg/addons/velero/velero.go

```diff
@@ -0,0 +1,121 @@
+package velero
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
+	"github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"gopkg.in/yaml.v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/kubeutils"
+	"github.com/replicatedhq/embedded-cluster/pkg/spinner"
+)
+
+const (
+	releaseName = "velero"
+	namespace   = "velero"
+)
+
+// Overwritten by -ldflags in Makefile
+var (
+	ChartURL  = "https://url"
+	ChartName = "name"
+	Version   = "v0.0.0"
+	VeleroTag = "v0.0.0"
+)
+
+var helmValues = map[string]interface{}{
+	"backupsEnabled":   false,
+	"snapshotsEnabled": false,
+	"deployNodeAgent":  true,
+	"nodeAgent": map[string]interface{}{
+		"podVolumePath": "/var/lib/k0s/kubelet/pods",
+	},
+	"image": map[string]interface{}{
+		"tag": VeleroTag,
+	},
+}
```

Comment:
Would we need to also install the AWS plugin to support s3 backups? 

Not entirely sure how to do it in the chart, but this look like something? https://github.com/vmware-tanzu/helm-charts/blob/main/charts/velero/values.yaml#L105-L119